docs/standard-library/http/encoding.md

```diff
@@ -53,13 +53,13 @@ model User {
   updatedAt: offsetDateTime; // rfc3339
 
   @encode(DateTimeKnownEncoding.rfc7231)
-  createdAtPretty: utcDateTime; // rfc7231
+  createdAtPretty: utcDateTime; // date-time
```

Comment:
This was correct before no?

Reply:
Currently no -- neither are really correct for the http library, as we are discussing in #2256.

But I suppose changing from one incorrect value to another isn't justified, so I will revert.

Reply:
well this one had an explicit `@encode` so wouldn't apply in #2256 discussion, no?